soccer_ipm/setup.py

```diff
@@ -0,0 +1,33 @@
+import glob
+
+from setuptools import setup
+
+package_name = 'soccer_ipm'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name + '/config',
+            glob.glob('config/*.yaml')),
+        ('share/' + package_name + '/launch',
+            glob.glob('launch/*.launch')),
+
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='florian',
```

Comment:
Assuming you want this to match the one in package.xml:

```suggestion
    maintainer='Florian Vahl',
```